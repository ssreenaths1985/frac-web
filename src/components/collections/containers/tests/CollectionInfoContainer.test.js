import React from "react";
import { shallow } from "enzyme";
import CollectionContainer from "../CollectionContainer";
const message = "No collections selected";

const collectionContainerProps = {
  history: {
    location: {
      pathname: "/collections",
      search: "",
      hash: "",
      state: null,
      key: "zgfeb7",
    },
  },
  handleClick: () => {
    console.log("click");
  },
};

let wrapped = shallow(<CollectionContainer {...collectionContainerProps}></CollectionContainer>);

describe("CollectionContainer", () => {
  it("should render the CollectionContainer Component correctly", () => {
    expect(wrapped).toMatchSnapshot();
  });
  it("renders the CollectionContainer children", () => {
    expect(wrapped.find("h1").text()).toEqual(message);
  });
});
