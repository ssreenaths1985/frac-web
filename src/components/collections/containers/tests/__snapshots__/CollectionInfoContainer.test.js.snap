// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CollectionContainer should render the CollectionContainer Component correctly 1`] = `
<Fragment>
  <div
    className="col-xs-12 col-sm-12 col-md-12 col-lg-12"
  >
    <HeaderSection
      handleClick={[Function]}
      history={
        Object {
          "location": Object {
            "hash": "",
            "key": "zgfeb7",
            "pathname": "/collections",
            "search": "",
            "state": null,
          },
        }
      }
    />
  </div>
  <div
    className="d-xl-flex d-lg-flex d-md-flex d-sm-none col-xs-12 col-sm-12 col-md-12 col-lg-12 custom-padding-1"
  >
    <ColumnOne
      getDeptValue={[Function]}
      handleClick={[Function]}
      history={
        Object {
          "location": Object {
            "hash": "",
            "key": "zgfeb7",
            "pathname": "/collections",
            "search": "",
            "state": null,
          },
        }
      }
    />
    <div
      className="col-xs-12 col-sm-12 col-md-10 col-lg-10 col-xl-10 bordered custom-full-height-4 custom-body-bg"
    >
      <div
        className="vertical-center d-none d-sm-block"
        id="emptyState"
      >
        <center>
          <h1
            className="pb-2"
          >
            No collections selected
          </h1>
          <p
            className="col-7 pb-2"
          >
            Please select a collection from the left side to see its details here
          </p>
          <img
            alt="empty list"
            src="../../img/empty/lists_empty.svg"
          />
        </center>
      </div>
    </div>
  </div>
</Fragment>
`;
